src/lib/draw/route/SplitRouteMode.svelte

```diff
@@ -25,40 +26,43 @@
   export let changeMode: (m: Mode) => void;
   export let eventHandler: EventHandler;
 
-  export function start() {}
+  export function start() {
+    // Use a fallback icon in case the image fails
+    $map.getCanvas().style.cursor = `url(${splitIcon}), crosshair`;
+  }
   export function stop() {
-    cursor = null;
+    $map.getCanvas().style.cursor = "inherit";
+    snappedCursor = null;
     snappedIndex = null;
   }
 
-  let cursor: Feature<Point> | null = null;
+  let snappedCursor: Feature<Point> | null = null;
```

Comment:
The behavior in this tool is hopefully intuitive. We always show the scissors icon where the user's cursor is. When they're close enough to snap to a route and make the action become valid, we draw a solid circle to show where the split will happen.

Reply:
This is super nice. Seems really intuitive to me and quite elegant